File: 프로그래머스/unrated/120808. 분수의 덧셈/분수의 덧셈.js
function solution(numer1, denom1, numer2, denom2) {
    let numer = numer1 * denom2 + numer2 * denom1;
    let denom = denom1 * denom2;
    for(let i = denom ; i > 1 ; i--){
        if(numer%i == 0 && denom%i == 0){
            numer /= i
            denom/= i
            break
        }
    }
    return [numer, denom];
}

File: 프로그래머스/unrated/120808. 분수의 덧셈/분수의 덧셈.py
def solution(numer1, denom1, numer2, denom2):
    numer = denom1*numer2 + denom2*numer1
    denom = denom1*denom2
    for i in range(denom, 1, -1):
        if numer%i == 0 and denom%i  == 0:
            numer = numer//i
            denom = denom//i
    answer = [numer, denom]
    return answer